samples/notebooks/dotnet/7-DALL-E-2.ipynb

```diff
@@ -0,0 +1,216 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Set up the environment, importing Semantic Kernel SDK and SkiaSharp, used to display images inline."
```

Comment:
Suggestion: Add a short intro paragraph similar to the other notebooks that explains what the purpose of the notebook is

```
# Image Generation Models with the Semantic Kernel
The following notebook shows how you can use image generation models like Dall-E with the Semantic Kernel. 
It also shows an end to end example how you can make use of embeddings to create a simple guessing game based off the text description of the image generated. 
```